packages/utilities/psammead-locales/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.11.0 | [PR#2178](https://github.com/bbc/psammead/pull/2178)  Update `ps` (Pashto) locale |
```

Comment:
Please could you briefly say what we've changed, e.g. `...to add relative timestamp support`?